core/ssh_keys.go

```diff
@@ -0,0 +1,106 @@
+package core
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/krystal/go-katapult"
+)
+
+type AuthSSHKey struct {
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitempty"`
+	Fingerprint string `json:"fingerprint,omitempty"`
+}
+
+type SSHKeysClient struct {
+	client   RequestMaker
+	basePath *url.URL
+}
+
+func NewSSHKeysClient(rm RequestMaker) *SSHKeysClient {
+	return &SSHKeysClient{
+		client:   rm,
+		basePath: &url.URL{Path: "/core/v1/"},
+	}
+}
+
+type sshKeysResponseBody struct {
+	SSHKey  *AuthSSHKey   `json:"ssh_key,omitempty"`
+	SSHKeys []*AuthSSHKey `json:"ssh_keys,omitempty"`
+}
+
+func (s *SSHKeysClient) List(
+	ctx context.Context,
+	ref OrganizationRef,
+) ([]*AuthSSHKey, *katapult.Response, error) {
+	qs := ref.queryValues()
+	u := &url.URL{Path: "organizations/_/ssh_keys", RawQuery: qs.Encode()}
+
+	body, resp, err := s.doRequest(ctx, "GET", u, nil)
+
+	return body.SSHKeys, resp, err
+}
+
+type AuthSSHKeyProperties struct {
+	// Name is the SSH keys name.
+	Name string `json:"name"`
+
+	// Key is the SSH public key.
+	Key string `json:"key"`
+}
+
+func (s *SSHKeysClient) Add(
+	ctx context.Context,
+	ref OrganizationRef,
+	properties AuthSSHKeyProperties,
+) (*AuthSSHKey, *katapult.Response, error) {
+	qs := ref.queryValues()
+	u := &url.URL{Path: "organizations/_/ssh_keys", RawQuery: qs.Encode()}
+
+	body, resp, err := s.doRequest(ctx, "POST", u, properties)
+
+	return body.SSHKey, resp, err
+}
+
+type SSHKeyRef struct {
+	ID string `json:"id"`
+}
+
+func (kr SSHKeyRef) queryValues() *url.Values {
+	v := &url.Values{}
+
+	v.Set("ssh_key[id]", kr.ID)
+
+	return v
```

Comment:
Nitpick: This can be simplified a bit:

```suggestion
	return &url.Values{"ssh_key[id]": []string{kr.ID}}
```